clang/test/SemaSYCL/sycl-restrict.cpp

```diff
@@ -1,11 +1,18 @@
-// RUN: %clang_cc1 -fcxx-exceptions -triple spir64 -fsycl-is-device -Wno-return-type -verify -fsyntax-only -std=c++17 %s
+// RUN: %clang_cc1 -fcxx-exceptions -triple spir64 -fsycl-is-device -Wno-return-type -verify -fsyntax-only -std=c++17 -DSYCL_STRICT %s
```

Comment:
What is `-DSYCL_STRICT` for?
I can't find any uses of this define in the test.

Reply:
It's a rudimentary change.
Removed.